pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -96,7 +100,30 @@ type RegistryPoll struct {
 	// Interval is used to determine the time interval between checks of the latest catalog source version.
 	// The catalog operator polls to see if a new version of the catalog source is available.
 	// If available, the latest image is pulled and gRPC traffic is directed to the latest catalog source.
-	Interval *metav1.Duration `json:"interval,omitempty"`
+	Interval     *metav1.Duration `json:"interval,omitempty"`
+	ParsingError string           `json:"-"`
```

Comment:
Does this field need to be exported?

Reply:
Yes, since we now want to use it outside of the `v1alpha1` package.

Reply:
Specifically in the catsrc sync loops where we'll check for this error before projecting it onto the status condition.